README.md

```diff
@@ -2,7 +2,7 @@
 
 # School library
 
-> In this project I made a person object following the decorator pattern, allowing the user to capitalize the person number, or to trim it
+> In this project, you can ran the aplication to start uploading books, persons and rentals using the comand line interface!
```

Comment:
- [ ] In your `README.md` file, it would be nice to include a command that shows how to install the dependencies (gems) used in the project.

Reply:
Hi, which dependencies? I only installed robocop but I thought that wasn't necessary since it was only for linters

Reply:
Hi @Gambit142, which dependencies should I add? I only installed Rubocop, but I didn't list it since it was only for linters

Reply:
Hi @lisandroseia ,

You don't have a Gemfile, so I would not be able to run `bundle install`. I am not talking about Rubocop. 👍 . Kindly include a `Gemfile`. A Gemfile is an equivalent of `Package.json` file. In Ruby, those dependencies are called `Gems`, thus the name `Gemfile`.

Reply:
okay @Gambit142, I never read about that, and I have no dependencies, so I will need to look up some information.
 I thought this wasn't necessary since my partner doesn't have it and no one requested it, but thanks for showing it up, I will add it today